packages/admin/src/common/toolbar/backbutton/ToolbarBackButton.styles.tsx

```diff
@@ -0,0 +1,45 @@
+import { Theme } from "@material-ui/core/styles";
+import { ArrowBack } from "@material-ui/icons";
+import { makeStyles } from "@material-ui/styles";
+import * as React from "react";
+
+import { useComponentThemeProps } from "../../../mui/useComponentThemeProps";
+
+export interface ToolbarBackButtonThemeProps {
+    backIcon?: React.ReactNode;
+}
+
+export type CometAdminToolbarBackButtonClassKeys = "root";
+
+export const useStyles = makeStyles<Theme, {}, CometAdminToolbarBackButtonClassKeys>(
+    () => ({
+        root: {
+            flex: 0,
+            display: "flex",
+            alignItems: "stretch",
+
+            "& [class*='ToolbarItem']": {
```

Comment:
This should probably be:
```suggestion
            "& [class*='CometAdminToolbarItem-root']": {
```

Reply:
true, both work but yours is more explicit